src/lib/plugins/moderation.js

```diff
@@ -98,10 +99,10 @@ module.exports.server = function (serv, settings) {
         if (!player.op) {
           player.op = true
 
-          player.chat(`§7§o[Server: Opped ${params[0]}]`)
-          return `Opped ${params[0]}`
+          player.chat(`§7§o[Server: Opped ${player.username}]`)
```

Comment:
Why [Server:.. ?
why not ${player.username || 'Server'

Reply:
done in latest commit